docs/source/CppUsage.md

```diff
@@ -495,4 +495,45 @@ needed to use unions.
 
 To use scalars, simply wrap them in a struct.
 
+## Dependence from C-locale {#flatbuffers_locale_cpp}
+Flatbuffers [grammar](@ref flatbuffers grammar) is based on ASCII
```

Comment:
I wasn't aware of this, and the fact that we need all this is very unfortunate. It be nice if we could parse floats correctly regardless of the locale settings on a build machine.. but that involves writing our own strtod? Are there any better solutions?
I've personally never understood locales.. the idea that code should parse things differently depending on the local language seems very error prone to me.